public/javascripts/main.js

```diff
@@ -0,0 +1,56 @@
+function getZip() {
+
+    $(document).click(function (e) {
+        e.preventDefault();
```

Comment:
What does this actually do? I've noticed that on failed searches (searching for "rrr" for example) I can't click anywhere.